### PR TITLE
feat: add support for encrypted signers ( passphrase )

### PR DESCRIPTION
**Pull request by azjezz**
needs https://github.com/symfony/mercure/pull/74

### CODE REVIEW

File: src/DependencyInjection/MercureExtension.php [hunk 1]
```diff
@@ -107,6 +107,8 @@ public function load(array $configs, ContainerBuilder $container)
                         $container->register($tokenFactory, LcobucciFactory::class)
                             ->addArgument($hub['jwt']['secret'])
                             ->addArgument($hub['jwt']['algorithm'])
+                            ->addArgument(null)
+                            ->addArgument($hub['jwt']['passphrase'])
```
Comment (azjezz) [inline src/DependencyInjection/MercureExtension.php:111]: should we throw if passphrase is provided by `symfony/mercure` version doesn't support passphrase? 🤔 

cc @dunglas

Comment (dunglas) [reply src/DependencyInjection/MercureExtension.php:111]: Yes, we should throw or bump the minimum version.

**Review by dunglas — APPROVED** [review]

File: src/DependencyInjection/MercureExtension.php (hunk 1 above)
Comment (azjezz) [reply src/DependencyInjection/MercureExtension.php:111]: okay, let's wait for a release on `symfony/mercure`, then we can bump the minimum here.

**Review by chalasr — APPROVED** [review]

**Comment by dunglas** [comment]
Thanks @azjezz!

**Pull request merged**